src/components/Calendar/doubleCalendar/styled/select.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { FONT_SIZE_HEADING_MEDIUM, FONT_SIZE_TEXT_LARGE } from '../../../../styles/fontSizes';
+import { PADDING_MEDIUM } from '../../../../styles/paddings';
+import attachThemeAttrs from '../../../../styles/helpers/attachThemeAttrs';
+import Select from '../../../Select';
+
+const StyledSelect = attachThemeAttrs(styled(Select))`
+    select {
```

Comment:
Here we are doing bad practices, we have to avoid doing introspection because it could give us problems. whats happen if in the future the Select change? this could break
Since we are changing almost all the Select styles then it is not feasible to use our Select component instead we use the native select.